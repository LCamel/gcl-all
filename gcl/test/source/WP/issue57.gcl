con N : Int { 0 â‰¤ N }
var r : Int

{:  
:}

{-
-}