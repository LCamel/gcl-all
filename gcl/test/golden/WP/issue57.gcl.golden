### Result

[]

### Source

con N : Int { 0 â‰¤ N }
var r : Int

{:  
:}

{-
-}

### Trace

TraceLog "      [ save ] Parsed"
TraceLog "      [ save ] Converted"
TraceLog "      [ save ] TypeChecked"
TraceLog "      [ save ] Swept"
